ios/build.sh

```diff
@@ -135,16 +142,15 @@ echo "Building KMEI..."
 rm -r $KMEI_BUILD_PATH/$CONFIG-universal 2>/dev/null
 xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -target KME-universal -configuration $CONFIG \
   $CODE_SIGN_IDENTITY $CODE_SIGNING_REQUIRED $DEV_TEAM
-assertDirExists "$BUILD_FRAMEWORK_PATH"
 
-if [ $BUILD_NUMBER ]; then
-  echo "Setting version number in KeymanEngine to $BUILD_NUMBER."
-  /usr/libexec/Plistbuddy -c "Set CFBundleVersion $BUILD_NUMBER" "$BUILD_FRAMEWORK_PATH/Info.plist"
-  /usr/libexec/Plistbuddy -c "Set CFBundleShortVersionString $BUILD_NUMBER" "$BUILD_FRAMEWORK_PATH/Info.plist"
+if [ $? -ne 0 ]; then
+  fail "KMEI build failed."
 fi
 
-rm -r "$APP_FRAMEWORK_PATH"
-cp -R "$BUILD_FRAMEWORK_PATH" "$APP_FRAMEWORK_PATH"
+assertDirExists "$BUILD_FRAMEWORK_PATH_UNIVERSAL"
+
+set_version "$BUILD_FRAMEWORK_PATH_UNIVERSAL" "KeymanEngine"
+set_version "$BUILD_FRAMEWORK_PATH_IOS"
```

Comment:
General question: is this script aiming to build both universal and iphone KeymanEngine frameworks as artifacts?

Reply:
Yes, I've put in prep work so that both will be available to developers.